pkg/katautils/config.go

```diff
@@ -102,6 +102,11 @@ type hypervisor struct {
 	DefaultBridges          uint32 `toml:"default_bridges"`
 	Msize9p                 uint32 `toml:"msize_9p"`
 	DisableBlockDeviceUse   bool   `toml:"disable_block_device_use"`
+	VirtioFS                bool   `toml:"enable_virtio_fs"`
```

Comment:
It would be useful to see something in the output of `kata-runtime kata-env` to show that virtio-fs is active. This saves users from having to find the correct configuration file (the runtime support multiple locations).

For examples, see #702, #937.